### PR TITLE
[ContentDialog] Update background smoke for HC

### PR DESCRIPTION
**Pull request by tashatitova**
Based on user feedback, some smoke is preferred in HC scenarios as well.

Before and after, aqua:
aqua-before
aqua-updated]

Before and after, desert:
desert-before

desert-updated

### CODE REVIEW

File: dev/CommonStyles/CalendarDatePicker_themeresources.xaml [hunk 1]
```diff
@@ -14,40 +14,55 @@
             <StaticResource x:Key="CalendarDatePickerForeground" ResourceKey="TextFillColorPrimaryBrush" />
             <StaticResource x:Key="CalendarDatePickerForegroundDisabled" ResourceKey="TextFillColorDisabledBrush" />
             <StaticResource x:Key="CalendarDatePickerCalendarGlyphForeground" ResourceKey="TextFillColorSecondaryBrush" />
+            <StaticResource x:Key="CalendarDatePickerCalendarGlyphForegroundPointerOver" ResourceKey="TextFillColorSecondaryBrush" />
```
Comment (ranjeshj) [inline dev/CommonStyles/CalendarDatePicker_themeresources.xaml:17]: Looks like Picker updates are unrelated. Can you undo the changes not related to content dialog ?

Comment (tashatitova) [reply dev/CommonStyles/CalendarDatePicker_themeresources.xaml]: my bad, updated. thanks!

**Review by marksfoster — APPROVED** [review]
